topic-gateway/src/test/java/io/sk8s/topic/gateway/TopicGatewayApplicationTests.java

```diff
@@ -24,9 +24,11 @@
 
 /**
  * @author Mark Fisher
+ * @author Thomas Risberg
  */
-@RunWith(SpringRunner.class)
-@SpringBootTest
+// Need a JUnit Kafka Rule to avoid failing when Kafka not available
```

Comment:
@trisberg did you create an issue for this? similar to https://github.com/markfisher/sk8s/issues/26

Reply:
Have created one - #39